cmd/certs/main.go

```diff
@@ -169,13 +169,14 @@ func loadCertificates(conf config, logger mflog.Logger) (tls.Certificate, *x509.
 	}
 
 	block, _ := pem.Decode(b)
-	if block == nil {
+	switch block {
```

Comment:
Using `if` in this case is fine, preferable even. What was linter's remark?

Reply:
Can you, please, send me the exact `golangci-lint` command to test this? The command we have in CI script (`golangci-lint run --no-config --disable-all --enable gosimple --enable errcheck --enable govet --enable unused --enable goconst --timeout 3m`) does not show me any problems with original code.

Reply:
This is no longer highlighted after updating the linter, so I've reverted back to original.
Maybe they fixed the linter in the update.
But when I use the original
```
        block, _ := pem.Decode(b)
	if block == nil {
		logger.Fatal("No PEM data found, failed to decode CA")
	}

	caCert, err = x509.ParseCertificate(block.Bytes)    <--- IDE warns here about possible nil pointer dereference even after checking for nil above
	if err != nil {
		return tlsCert, caCert, errors.Wrap(errFailedCertDecode, err)
	}
```

Reply:
@dborovcanin Should I leave this as it was(original code) or change to switch statement?

Reply:
Let's keep it as it is.